src/server/routes/signin.js

```diff
@@ -38,17 +38,21 @@ export default async function signin(req, res) {
     // according to RFC 2821, but in practice this causes more problems than
     // it solves. We treat email addresses as all lower case. If anyone
     // complains about this we can make strict RFC 2821 compliance an option.
-    const email = req.body.email?.toLowerCase() ?? null
+    const email = req.body.email.toLowerCase()
```

Comment:
We skip `?`  here, which means we always expect `email` to be always supplied in the `POST` request. Are we sure it won't create any side effects 🙇🏽‍♂️